src/main/features/dashboard/views/macro/responseStatus.njk

```diff
@@ -4,6 +4,25 @@
   <a href="{{ ResponsePaths.summaryPage.evaluateUri({ externalId: externalId }) }}">{{ t('View full response') }}</a>
 {% endmacro %}
 
+{% macro responseStatusForDefendantBasedOnState(defendantClaimState, claim, url, isAfter4pm) %}
```

Comment:
Please rename `defendantClaimState`. I do think it is a bad name because claim state is the same for everyone, it is party agnostic.